collada/scene.py

```diff
@@ -419,7 +430,7 @@ def load( collada, node, localscope ):
             except DaeError as ex:
                 collada.handleError(ex)
 
-        return Node(id, children, transforms, xmlnode=node)
+        return Node(id, children, transforms, xmlnode=node, name)
```

Comment:
you shouldn't have positional arguments after named arguments, so use name=name

Reply:
Ok, thanks!